src/commands/delete-excsessive-pvc

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+#set -x
+
+readonly PVC_SELECTOR=${1:-app=collection-sumologic-fluentd-logs}
+readonly HPA_NAME=${2:-collection-sumologic-fluentd-logs}
+readonly NAMESPACE=${3:-sumologic}
```

Comment:
Maybe we shouldn't use default values, because the question is why `logs` not `metrics`? 🤔

Reply:
Yeah, that's actually a good idea, will change that.

Reply:
@perk-sumo  so?

Reply:
Done.d